docs/source/release_notes.rst

```diff
@@ -2,6 +2,16 @@
  Release History
 =================
 
+1.6.1 (2021-02-26)
+==================
+
+Fixes
+-----
+
+* Added a missing name argument to AreaDetector documentation
+* Correctly propagate changes to the default ``connection_timeout`` setting
+  via ``set_defaults``, instead of always defaulting to 1.0 seconds.
```

Comment:
```suggestion
  via ``set_defaults``, instead of always defaulting to 1.0 second.
```

Reply:
It's a bit counter-intuitive, but I think the way I have it is grammatically correct. 

Only an integer 1 can be singular, while all floating point numbers are plural. So it would have to either be "1 second" or "1.0 seconds"... though I have no strong preference to which of the two to use. I picked 1.0 because that is exactly as it is in the code.

Reply:
That's a good point. I think the release notes are for humans, not computers, maybe having "1 second" makes sense here. I'll update my suggestion.